src/moz_build_info.js

```diff
@@ -20,7 +23,11 @@ export function archiveFiletype(os) {
  */
 export function archiveFileSuffix(product, os) {
   let ospart = (product === 'firefox' && os === 'mac64') ? 'mac' : os;
-  return `${ospart}.${archiveFiletype(os)}`;
+  if (product.indexOf('emulator') !== -1) {
+    // emulator doen't have os in filename
+    return `${archiveFiletype(os, product)}`;
```

Comment:
We don't need a template string if it's just the one value (ie you could simply return `archiveFiletype(os, product)`. Also since we're calling this method regardless of whether the product is an emulator or not, why not `let filetype = archiveFiletype(os, product)`?